src/allmydata/test/cli/test_invite.py

```diff
@@ -19,7 +19,8 @@
 from ..common_util import run_cli
 from ..no_network import GridTestMixin
 from .common import CLITestMixin
-from .wormholetesting import IWormhole, MemoryWormholeServer, TestingHelper, memory_server
+from .wormholetesting import MemoryWormholeServer, TestingHelper, memory_server
+from wormhole._interfaces import IWormhole
```

Comment:
I have tried this where I use ```from .wormholetesting import *``` and declaring ```__all__ = ['MemoryWormholeServer', 'TestingHelper', 'memory_server', 'IWormhole']``` in ```wormholetesting.py```. However, if I do this it will fail the ```codechecks``` test tox will run with the following message:
```
src/allmydata/test/cli/test_invite.py:22:1: F403 'from .wormholetesting import *' used; unable to detect undefined names
src/allmydata/test/cli/test_invite.py:31:46: F405 'IWormhole' may be undefined, or defined from star imports: .wormholetesting
...
```
Looking up the error it seems like ```codechecks``` test wants explicit imports to files.

Reply:
> I have tried this where I use `from .wormholetesting import *`

Why this part?

Reply:
`__all__` does a few things.  Defining what `import *` means is one of them - but in this case, that's not what we're trying to do.  We're just trying to convince mypy that `IWormhole` is allowed to come from `wormholetesting` even though it isn't defined there.

So, leave the `from .wormholetesting ...` line alone and only add the `__all__`.

Reply:
I see. Thanks I have made the changes.